### ISSUE TITLE
lcov report error with Webpack template of Vue.js

### ISSUE THREAD

**Issue by wxsms** [mattlewis92/karma-coverage-istanbul-reporter#32]
Thanks for making this nice reporter. I've been using it for a long time. 👍

## Note

This error only happen with [vue-loader](https://github.com/vuejs/vue-loader)@^13.3.0. versions before that works great.

## Reproduce

Init a project with [vue-cli](https://github.com/vuejs/vue-cli):

```
$ vue-init webpack
```

(the installed vue-loader version is ^13.3.0 in the latest template)

in the unit test section, select karma+mocha runner.

After project init, change the coverage reporter to this lib by:

```javascript
// test/unit/karma.conf.js
 
module.exports = function (config) {
  config.set({
    // ...
    reporters: ['spec', 'coverage-istanbul'],
    coverageIstanbulReporter: {
      reports: ['lcov', 'text-summary'],
      dir: path.join(__dirname, 'coverage'),
      fixWebpackSourcePaths: true,
      skipFilesWithNoCoverage: true,
      'report-config': {
        lcov: {
          subdir: '.'
        }
      }
    }
  })
}
```

ok. let's run `npm test`. it ends with no error.

Then, open the lcov report at `test/unit/coverage/lcov-report/index.html`, click `HelloWorld.vue`, **the coverage report inside this file is an error message**:

```
Unable to lookup source: /Users/wxsm/githome/vue-webpack-test/src/components/src/components/HelloWorld.vue(ENOENT: no such file or directory, open '/Users/wxsm/githome/vue-webpack-test/src/components/src/components/HelloWorld.vue')
Error: Unable to lookup source: /Users/wxsm/githome/vue-webpack-test/src/components/src/components/HelloWorld.vue(ENOENT: no such file or directory, open '/Users/wxsm/githome/vue-webpack-test/src/components/src/components/HelloWorld.vue')
    at Context.defaultSourceLookup [as sourceFinder] (/Users/wxsm/githome/vue-webpack-test/node_modules/istanbul-lib-report/lib/context.js:15:15)
    at Context.getSource (/Users/wxsm/githome/vue-webpack-test/node_modules/istanbul-lib-report/lib/context.js:74:17)
    at Object.annotateSourceCode (/Users/wxsm/githome/vue-webpack-test/node_modules/istanbul-reports/lib/html/annotator.js:172:38)
    at HtmlReport.onDetail (/Users/wxsm/githome/vue-webpack-test/node_modules/istanbul-reports/lib/html/index.js:217:39)
    at LcovReport.(anonymous function) [as onDetail] (/Users/wxsm/githome/vue-webpack-test/node_modules/istanbul-reports/lib/lcov/index.js:24:24)
    at Visitor.(anonymous function) [as onDetail] (/Users/wxsm/githome/vue-webpack-test/node_modules/istanbul-lib-report/lib/tree.js:34:30)
    at ReportNode.Node.visit (/Users/wxsm/githome/vue-webpack-test/node_modules/istanbul-lib-report/lib/tree.js:123:17)
    at /Users/wxsm/githome/vue-webpack-test/node_modules/istanbul-lib-report/lib/tree.js:116:23
    at Array.forEach ()
    at visitChildren (/Users/wxsm/githome/vue-webpack-test/node_modules/istanbul-lib-report/lib/tree.js:115:32)
    at ReportNode.Node.visit (/Users/wxsm/githome/vue-webpack-test/node_modules/istanbul-lib-report/lib/tree.js:126:5)
    at Tree.visit (/Users/wxsm/githome/vue-webpack-test/node_modules/istanbul-lib-report/lib/tree.js:158:20)
    at /Users/wxsm/githome/vue-webpack-test/node_modules/istanbul-api/lib/reporter.js:84:18
    at Array.forEach ()
    at Reporter.write (/Users/wxsm/githome/vue-webpack-test/node_modules/istanbul-api/lib/reporter.js:82:35)
    at browsers.forEach.browser (/Users/wxsm/githome/vue-webpack-test/node_modules/karma-coverage-istanbul-reporter/src/reporter.js:82:16)
    at Array.forEach ()
    at Collection.forEach (/Users/wxsm/githome/vue-webpack-test/node_modules/karma/lib/browser_collection.js:93:21)
    at CoverageIstanbulReporter.onRunComplete (/Users/wxsm/githome/vue-webpack-test/node_modules/karma-coverage-istanbul-reporter/src/reporter.js:38:14)
    at Server. (/Users/wxsm/githome/vue-webpack-test/node_modules/karma/lib/events.js:13:22)
    at emitTwo (events.js:130:20)
    at Server.emit (events.js:213:7)
```

I think the problem is at `/vue-webpack-test/src/components/src/components/HelloWorld.vue`, looks like the reporter is looking for a wrong dir, which should be `/vue-webpack-test/src/components/HelloWorld.vue`.

**Comment by mattlewis92** [comment]
Hey there, thanks for the clear steps to reproduce the problem, that was really helpful in tracking down the problem quickly 😄  I've just published `1.3.1`, please can you give it a go and LMK if it works ok for you?

**Comment by wxsms** [comment]
Yes, it works now. just tried the latest 1.3.3.

Thank you very much!
